frontend/src/components/Documentation.vue

```diff
@@ -290,14 +290,32 @@
                 ). To clear the RNA levels, select the
                 <i>None</i>
                 option in the drop down list. RNA levels are available for both 2D and 3D Map
-                Viewer. The gene expression levels used are obtained from the file
-                <b>rna_tissue_hpa.tsv.zip</b>
-                in the
+                Viewer. The gene expression levels used are obtained from the
                 <a href="https://www.proteinatlas.org/about/download" target="_blank">
                   download page
                 </a>
                 of the Human Protein Atlas.
               </p>
+              <p>
+                The reaction presence was initially computed as a number between 0 and 100. The
+                value is computed by creating bootstraps from the single-cell RNA-Seq data from the
+                <a href="https://www.proteinatlas.org/about/releases#21.0" target="_blank"
+                  >Human Protein Atlas - Single Cell Type Section</a
+                >, pooling those into RNA-Seq profiles and applying the ftINIT algorithm. A high
+                value >=0.9 means that there is high confidence that the reaction is present,
```

Comment:
The value needs to be corrected in light of @johan-gson's comment above, and the HTML character should be used:
```suggestion
                value &#8805;0.99 means that there is high confidence that the reaction is present,
```

Reply:
Fixed in 1c9047e